test/soak/serve_hostnames/serve_hostnames.go

```diff
@@ -44,6 +44,7 @@ var (
 	podsPerNode    = flag.Int("pods_per_node", 1, "Number of serve_hostname pods per node")
 	upTo           = flag.Int("up_to", 1, "Number of iterations or -1 for no limit")
 	maxPar         = flag.Int("max_par", 500, "Maximum number of queries in flight")
+	gke            = flag.String("gke", "", "Taget GKE cluster with context gke_{project}_{zone}_{cluster-name}")
```

Comment:
How do I know if it is a GCE cluster? Here, I assume that if a `--gke` flag is given then it is a GKE cluster, otherwise it is a GCE cluster.